modules/launcher/src/test/java/org/mule/module/launcher/MuleApplicationClassLoaderTestCase.java

```diff
@@ -98,6 +102,32 @@ public void cleanUp()
     }
 
     @Test
+    public void leakCleanerIsCreatedByCorrectClassLoaderApp() throws Exception
```

Comment:
Can you move these test into a new test class? (current test class has a setup method that builds a complete classloader seting up folders and stuff)